src/ImageSharp/Formats/Jpeg/PdfJsPort/Components/PdfJsScanDecoder.cs

```diff
@@ -705,23 +718,21 @@ private int ReceiveAndExtend(int length, Stream stream)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I suggest to remove AggressiveInlining here, might help producing cleaner profiler results.